addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -151,7 +151,7 @@ function factory(dependencies) {
             if (!this.isClientRtcCompatible) {
                 return;
             }
-            if (!this._peerConnections[sender] && (!channelId || channelId !== this.channel.id)) {
+            if (!this._peerConnections[sender] && (!channelId || !this.channel || channelId !== this.channel.id)) {
```

Comment:
grep to make sure there are no such other occurrences in the code (`channel.id`, `channelId` and `channel_id` which was the previous name)

Reply:
at least in the rtc related fields